cernopendata/modules/fixtures/data/records/delphi-documentation-manuals.json

```diff
@@ -0,0 +1,205 @@
+[
+  {
+    "abstract": {
+      "description": "Detailed description of the DELPHI conditions database contents, and the tools coming with it."
```

Comment:
Cosmetics: Perhaps you may want to enrich the abstract to say that CARGO describes the database management package, DDAP describes the detection description application package, and DPLOT describes the database graphics?

Reply:
Valid point. Done.